tests/test_deserialization.py

```diff
@@ -284,18 +283,29 @@ class MyBoolean(fields.Boolean):
     ])
     def test_boolean_field_deserialization_with_custom_truthy_values_invalid(
             self, in_val):
-        class MyBoolean(fields.Boolean):
-            truthy = set(['yep'])
-        field = MyBoolean()
+        field = fields.Boolean(truthy=('yep',))
```

Comment:
Can you please leave existing tests unchanged and add new ones separately, because existing functionality is part of public interface (and thus need to be kept functioning).